### DISCUSSION TITLE
How to get the resolution of a case?

### DISCUSSION THREAD

**Discussion by xihaoxie-ltg** [freelawproject/courtlistener#3973] [Q&A]
In bulk data, how to know the resolution (e.g., Claimant Win / Defendant Win) of a specific case?

**Comment by mlissner** [comment]
No. That's a hard question and we've never found a good way to figure it out. For federal cases, you *might* be able to find something in the FJC IDB, but I haven't checked. I'd be curious what you discover if you take a look.

**Reply by xihaoxie-ltg** [reply]
Let's me take a look at it. Thanks!
BTW, it seems that records in buld data are opinions. Is it possible to get complaints instead of opinions?

**Reply by mlissner** [reply]
The complaints are not in the bulk data, but are available via the APIs, via Internet Archive, or via storage.courtlistener.com. Get in touch via our contact form and we can discuss access options.
